plugins/processors/noise/README.md

```diff
@@ -0,0 +1,79 @@
+# Noise Processor
+
+The *Noise* processor is used to add noise to numerical field values. For each field a noise is generated using a defined probability densitiy function and added to the value. The function type can be configured as _Laplace_, _Gaussian_ or _Uniform_.
+Depending on the function, various parameters need to be configured:
+
+## Configuration
+
+Depending on the choice of the distribution function, the respective parameters must be set. Default settings are `noise_type = "laplacian"` with `mu = 0.0` and `scale = 1.0`:
+
+```toml
+[[processors.noise]]
+    ## Specified the type of the random distribution.
+    ## Can be "laplacian", "gaussian" or "uniform".
+    # type = "laplacian
+
+    ## Center of the distribution.
+    ## Only used for Laplacian and Gaussian distributions.
+    # mu = 0.0
+
+    ## Scale parameter for the Laplacian or Gaussian distribution
+    # scale = 1.0
+
+    ## Upper and lower bound of the Uniform distribution
+    # min = -1.0
+    # max = 1.0
+
+    ## Apply the noise only to numeric fields matching the filter criteria below.
+    ## Excludes takes precedence over includes.
+    # include_fields = []
+    # exclude_fields = []
```

Comment:
The formatting got mangled here. Please make this to start with two spaces only.